packages/react-components/react-rating-preview/src/components/RatingDisplay/useRatingDisplay.tsx

```diff
@@ -61,7 +61,7 @@ export const useRatingDisplay_unstable = (
     }),
     countText: slot.optional(props.countText, {
       renderByDefault: count !== undefined,
-      defaultProps: { children: count?.toLocaleString(), id: countTextId, 'aria-hidden': true },
+      defaultProps: { children: count && Math.round(count).toLocaleString(), id: countTextId, 'aria-hidden': true },
```

Comment:
When `count` is `0` this will return `false`. Previously it returned `"0"`.

Reply:
Good callout. I've updated this to check for this case and it also has the added benefit of preventing negative values from rendering